malariagen_data/pf7.py

```diff
@@ -0,0 +1,291 @@
+        for var_name in "POS", "CHROM":
+            z = root[f"variants/{var_name}"]
+            var = da_from_zarr(z, inline_array=inline_array, chunks=chunks)
+            coords[f"variant_{var_names_for_outputs[var_name]}"] = [DIM_VARIANT], var
+
+        z = root["samples"]
+        sample_id = da_from_zarr(z, inline_array=inline_array, chunks=chunks)
+        coords["sample_id"] = [DIM_SAMPLE], sample_id
+        return coords
+
+    def add_data_vars(self, root, inline_array, chunks, var_names_for_outputs):
+        data_vars = dict()
+
+        # variant_allele
+        ref_z = root["variants/REF"]
+        alt_z = root["variants/ALT"]
+        ref = da_from_zarr(ref_z, inline_array=inline_array, chunks=chunks)
+        alt = da_from_zarr(alt_z, inline_array=inline_array, chunks=chunks)
+        variant_allele = da.concatenate([ref[:, None], alt], axis=1)
+        data_vars["variant_allele"] = [DIM_VARIANT, DIM_ALLELE], variant_allele
+
+        # other default variant values
+        for var_name in ["FILTER_PASS", "is_snp", "numalt", "CDS"]:
+            z = root[f"variants/{var_name}"]
+            var = da_from_zarr(z, inline_array=inline_array, chunks=chunks)
+            if var_name in var_names_for_outputs.keys():
+                var_name = var_names_for_outputs[var_name]
+            data_vars[f"variant_{var_name}"] = [DIM_VARIANT], var
+
+        # call arrays
+        gt_z = root["calldata/GT"]
+        ad_z = root["calldata/AD"]
+        call_genotype = da_from_zarr(gt_z, inline_array=inline_array, chunks=chunks)
+        call_ad = da_from_zarr(ad_z, inline_array=inline_array, chunks=chunks)
+        data_vars["call_genotype"] = (
+            [DIM_VARIANT, DIM_SAMPLE, DIM_PLOIDY],
+            call_genotype,
+        )
+        data_vars["call_AD"] = ([DIM_VARIANT, DIM_SAMPLE, DIM_ALLELE], call_ad)
+
+        return data_vars
+
+    def _add_extended_data(self, root, inline_array, chunks, data_vars):
+
+        subset_extended_variants = self.extended_variant_fields
+        subset_extended_calldata = self.extended_calldata_variables
+
+        for var_name in subset_extended_calldata:
+            z = root[f"calldata/{var_name}"]
+            var = da_from_zarr(z, inline_array=inline_array, chunks=chunks)
+            data_vars[f"call_{var_name}"] = (
+                subset_extended_calldata[var_name],
+                var,
+            )
+
+        for var_name in subset_extended_variants:
+            z = root[f"variants/{var_name}"]
+            field = da_from_zarr(z, inline_array=inline_array, chunks=chunks)
+            data_vars[f"variant_{var_name}"] = (
+                subset_extended_variants[var_name],
+                field,
+            )
+        return data_vars
+
+    def variant_calls(self, extended=False, inline_array=True, chunks="native"):
+        """Access variant sites, site filters and genotype calls.
+        Parameters
+        ----------
+        inline_array : bool, optional
```

Comment:
Missing documentation of `extended` parameter.

Reply:
Added this in and just improved documentation in general